apps/offlineevents/views.py

```diff
@@ -10,3 +19,102 @@ class OfflineEventDetailView(module_views.ItemDetailView):
     @property
     def project(self):
         return self.get_object().project
+
+
+class OfflineEventMgmtView(DashboardBaseMixin,
+                           rules_mixins.PermissionRequiredMixin,
+                           generic.ListView):
+    model = models.OfflineEvent
+    template_name = 'meinberlin_offlineevents/offlineevent_mgmt_list.html'
+    permission_required = 'a4projects.add_project'
```

Comment:
maybe we should think about introducing a predicate `a4projects.moderate_project` or `a4projects.edit_project` for all the project dashboard related views.
we could make that predicate intelligent, so that it may accept an `Item` (or anything with a `.project` property), or a `Project` itself, or an `Organisation` and checks if the user is an initiator or in the projects moderator list